src/System.Private.ServiceModel/tools/CertificateCleanup/Program.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+
+namespace System.Private.ServiceModel.Tests.Tools
+{
+    class Program
+    {
+        private const string ClientCertificateSubject = "WCF Client Certificate";
+        private const string CertificateIssuer = "DO_NOT_TRUST_WcfBridgeRootCA";
+
+        static bool s_debug = false; 
+
+        public static void Main(string[] args)
+        {
+            // Parse arguments 
+            bool clean = false;
+
+            for (int i = 0; i < args.Length; i++)
+            {
+                if (string.Compare("--clean", args[i], StringComparison.OrdinalIgnoreCase) == 0)
+                {
+                    clean = true; 
```

Comment:
Nit: I'd probably make clean the default and -preview the way to say "show me what you would do".   Not critical.